doc/dev/find_sdk_in_cli_command.md

```diff
@@ -0,0 +1,88 @@
+How to Find the SDK in CLI 
+======
+
+This article aims to provide a guide for customer to find the sdk in Azure CLI.
+
+Find SDK function in cli command is divided into the following steps:
+1. Determine the command module
+2. Search its definition in `command.py`
+3. Find the corresponding SDK
+4. Find the function in Python SDK or Doc
+
+## Determine the command module
+
+Before looking for the SDK, please determine the command module you want to find. You can find it through the keyword in the CLI and the corresponding way [here](https://github.com/Azure/azure-cli/tree/dev/src/azure-cli/azure/cli/command_modules).
+
+Such as, if you want to find the module about `az resource group create`, you could find [`resource` folder](https://github.com/Azure/azure-cli/tree/dev/src/azure-cli/azure/cli/command_modules/resource) in above link.
+
+## Search its definition in `command.py`
+
+After entering the link, a command module folder generally contains the following files. You can find the command in `command.py`.
+```
+└─command module folder
+    └─ _help.py    # Store help information
+    └─ command.py    # Store command definition
+    └─ custom.py    # Store Custom method
+    └─ _params.py   # Store parameters information
+    └─....
+```
+Still the above sample `az resource group create`, You can find its definition on line 222 from [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/resource/commands.py#L222).
+
+## Find the corresponding SDK
+```python
+with self.command_group('group', resource_group_sdk, resource_type=ResourceType.MGMT_RESOURCE_RESOURCES) as g:
+    g.command('delete', 'begin_delete', supports_no_wait=True, confirmation=True)
+    g.show_command('show', 'get')
+    g.command('exists', 'check_existence')
+    g.custom_command('list', 'list_resource_groups', table_transformer=transform_resource_group_list)
+    g.custom_command('create', 'create_resource_group')
+    g.custom_command('export', 'export_group_as_template')
+    g.generic_update_command('update', custom_func_name='update_resource_group', custom_func_type=resource_custom)
+    g.wait_command('wait')
+```
```

Comment:
the definition of `resource_group_sdk` in this file gives the package name:
https://github.com/Azure/azure-cli/blob/a0057a0e4bbca7d5fd8563a7a78e78f4f68b1848/src/azure-cli/azure/cli/command_modules/resource/commands.py#L121-L125